cli/src/commands/extensions/extension.rs

```diff
@@ -141,8 +141,13 @@ impl Extension {
         Extension::try_from(extension_path(name)?)
     }
 
-    /// Return the path to this extension's entry point.
+    /// Return the path to this extension.
     pub fn path(&self) -> PathBuf {
+        self.path.clone()
+    }
+
+    /// Return the path to this extension's entry point.
+    pub fn entry_point_path(&self) -> PathBuf {
```

Comment:
```suggestion
    pub fn entry_point(&self) -> PathBuf {
```

Probably be fine to omit the type here? Path is the only representation we can return for entrypoint here anyway and that just makes things a little more concise.

Reply:
It would conflict with the other `entry_point` function at line 66, which returns the filename of the entry point (i.e. what is in the manifest). I agree the name is not the best.

Reply:
We could make both more explicit: `entry_point_directory` and `entry_point_filename`. What do you think?

Reply:
It seems like `entry_point` is only ever used to `join` with the manifest path? In that case having the `entry_point` method as the full path might actually clean things up?